format/transform.go

```diff
@@ -21,45 +21,77 @@ const (
 	// TransformRemoveTrailingNewlines removes extra newlines following
 	// sequence-like forms, so that parentheses are written on the same
 	// line. For example,
+	//
 	//   (foo bar
 	//    )
+	//
 	// becomes
+	//
 	//   (foo bar)
+	//
 	TransformRemoveTrailingNewlines
 
 	// TransformFixDefnArglistNewline moves the arg vector of defns to the
 	// same line, if appropriate:
+	//
 	//   (defn foo
 	//     [x] ...)
+	//
 	// becomes
+	//
 	//   (defn foo [x]
 	//     ...)
+	//
 	// if there's no newline after the arg list.
 	TransformFixDefnArglistNewline
 
 	// TransformFixDefmethodDispatchValNewline moves the dispatch-val of a
 	// defmethod to the same line, so that
+	//
 	//   (defmethod foo
 	//     :bar
 	//     [x] ...)
+	//
 	// becomes
+	//
 	//   (defmethod foo :bar
 	//     [x] ...)
+	//
 	TransformFixDefmethodDispatchValNewline
 
 	// TransformRemoveExtraBlankLines consolidates consecutive blank lines
 	// into a single blank line.
 	TransformRemoveExtraBlankLines
 
+	// TransformFixIfNewlineConsistency adds a newline before the 'true' arm of
+	// an if expression if the 'false' arm has a newline before it. This:
```

Comment:
I don't think this description captures this behavior:

```clojure
(if foo?
  "a" "b")

; becomes

(if foo?
  "a"
  "b")
```

Maybe you wrote this before handling that case?